sql/files/defaultdata/compare/compare.cc

```diff
@@ -72,30 +73,39 @@ FILE *openfeedback(const char *feedbackdir, const char *feedback, const char *wh
 	return res;
 }
 
+bool equal_case_insensitive(std::string a, std::string b)
+{
+	for(size_t i=1; i<a.length(); i++) a[i] = tolower(a[i]);
+	for(size_t i=1; i<b.length(); i++) b[i] = tolower(b[i]);
```

Comment:
nit: notice that calling ```tolower(char)``` is actually undefined behaviour see [the reference](https://en.cppreference.com/w/cpp/string/byte/tolower). And you probably want to write ```std::tolower```.

Reply:
Thanks, should be fixed now.